content/en/docs/concepts/containers/images.md

```diff
@@ -44,6 +44,16 @@ There are additional rules about where you can place the separator
 characters (`_`, `-`, and `.`) inside an image tag.  
 If you don't specify a tag, Kubernetes assumes you mean the tag `latest`.
 
+## Image pull per runtime class
+
+In K8s v1.29.0-alpha.3, CRI changes needed to support image pulls based on pod runtime class
+has been introduced. It is under feature gate `RuntimeClassInImageCriApi` and off by default.
+
+When the image pull per runtime class is GA, container images will be referenced as a tuple of
+(imageName, runtimeHandler) instead of just the imageName or image digest. This will be helpful
+in pulling suitable images for running VM based containers.
+Container runtimes may or may not add support for this feature.
```

Comment:
```suggestion
{{< feature-state for_k8s_version="v1.25" state="stable" >}}
Kubernetes includes alpha support for performing image pulls based on the RuntimeClass of a Pod.

If you enable the `RuntimeClassInImageCriApi` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/),
the kubelet references container images by a tuple of (image name, runtime handler) rather than just the
image name or digest. Your {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
may adapt its behavior based on the selected runtime handler. Provided your container runtime does include
support, this extra information passed via {{< glossary_tooltip text="CRI" term_id="cri" >}} is useful
if you mix low level runtime based on virtualization with runtimes based on Linux (cgroup and namespace) process isolation. 
```

Reply:
@sftim I am not sure I understand what you mean by "if you mix low level runtime based on virtualization with runtimes based on Linux (cgroup and namespace) process isolation.".

Reply:
Changed to:

Reply:
The feature is only really relevant if you use different ways to run containers, on the same node. For example, if everything is gVisor on a node, you don't need this feature. Or if everything is Kata + Firecracker.